### ISSUE TITLE
best miou 65.2

### ISSUE THREAD

**Issue by dongbo811** [lxtGH/PFSegNets#12]
It's a good job! I use 8x V100 (32GB) and follow the readme to run.
Why I can't reproduce the score in the paper? 
the best score as follows:
[val loss 0.05672], [acc 0.98903], [acc_cls 0.76118], [mean_iu 0.65289], [fwavacc 0.97973], [epoch 15],

**Comment by lxtGH** [comment]
ref to this issue setting. https://github.com/lxtGH/PFSegNets/issues/5.

**Comment by dongbo811** [comment]
Thx a lot!
I have try three times.
The mean miou is nearly 65.5. 
Could please help me to reproduce?

**Comment by lxtGH** [comment]
what is your cropped size of images?

**Comment by dongbo811** [comment]
follow the readme, and without any changes for the code.

**Comment by purse1996** [comment]
In fact, the code tests the mIou in the overlap patches on the validation set. Is it a reasonable way?

**Comment by dongbo811** [comment]
It doesn't seem that I can reproduce it with the weight provided by the author, but it seems that I can't reach it with my own training.
